c/src/atom.rs

```diff
@@ -1080,14 +1130,25 @@ pub extern "C" fn bindings_add_var_binding(bindings: *mut bindings_t, var: atom_
     }
 }
 
+/// @brief Checks if a `bindings_t` contains no associations
+/// @ingroup matching_group
+/// @param[in]  bindings  A pointer to the `bindings_t` to inspect
+/// @return `true` is the Bindings frame contains no associations, and `false` otherwise
+///
 #[no_mangle]
 pub extern "C" fn bindings_is_empty(bindings: *const bindings_t) -> bool{
     let bindings = unsafe{ &*bindings }.borrow();
     bindings.is_empty()
 }
 
-/// Returns the atom bound to the supplied variable in the bindings.  Returns a NULL atom_ref if the
-/// variable is not present.
+/// @brief Returns the atom bound to the supplied variable name in the `bindings_t`
+/// @ingroup matching_group
+/// @param[in]  bindings  A pointer to the `bindings_t` to inspect
+/// @param[in]  var_name  A NULL-terminated C-style string containing the name of the variable
+/// @return The `atom_t` representing the atom that corresponds to the specified variable, or a NULL `atom_ref_t` if the variable is not present.
+/// @note The caller must take ownership responsibility for the returned `atom_t`, if it is not NULL
+///
+//TODO: Question for Vitaly: Would be be better off here returnin an actual atom ref, so the caller can clone it if needed?
```

Comment:
It is not always possible to return the reference to the atom from Rust because sometimes atoms are constructed on the fly. One major case is an atom which is constructed by resolving its sub-variables via same instance of `Bindings`.